go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/distribution/distribution/v3
 go 1.18
 
 require (
+	cloud.google.com/go/storage v1.22.1
```

Comment:
the latest version is v1.25.0 now. 

https://pkg.go.dev/cloud.google.com/go/storage?tab=versions.

Reply:
Taking a look at the release notes, there are no security fixes in later versions. Are we missing functionality at this level that we need?

Reply:
I've updated it in a separate commit. So far it doesn't seem to cause any pain going from v1.22.1 to v1.25.0. If we want to drop it back to v1.22.1 for some reason I can back out the commit.

Reply:
If you're happy to do the work that's fine, just wondered if it was worth the effort :)